.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: GitHub Discussions
+    url: https://github.com/retrospring/retrospring/discussions
+    about: Please ask and answer questions here.
```

Comment:
funny that.  I guess it would be nice to give some examples here when you would want do that